### ISSUE TITLE
Variable explorer

### ISSUE THREAD

**Issue by dittosayan** [spyder-ide/spyder#9352]
### What steps will reproduce the problem?

Loaded an image and clicked on the variable explorer. Since image is shown as an array it should show the entire array, but instead it is crashing.

### Traceback
```python-traceback
  File "/usr/lib/python3/dist-packages/spyder/widgets/variableexplorer/collectionseditor.py", line 482, in createEditor
    if not editor.setup_and_check(value, title=key, readonly=readonly):
  File "/usr/lib/python3/dist-packages/spyder/widgets/variableexplorer/arrayeditor.py", line 620, in setup_and_check
    self.data.flags.writeable = True
ValueError: cannot set WRITEABLE flag to True of this array
```

## Versions

* Spyder version: 3.3.2 
* Python version: 3.7.3
* Qt version: 5.12.2
* PyQt5 version: 5.12.1
* Operating System: Linux 5.0.0-13-generic

### Dependencies

```
pyflakes >=0.6.0  :  2.0.0 (OK)
pycodestyle >=2.3 :  2.4.0 (OK)
pygments >=2.0    :  2.3.1 (OK)
sphinx >=0.6.6    :  1.8.5 (OK)
rope >=0.9.4      :  0.10.5 (OK)
jedi >=0.9.0      :  0.13.2 (OK)
psutil >=0.3      :  5.5.1 (OK)
nbconvert >=4.0   :  5.4.0 (OK)
pandas >=0.13.1   :  0.24.2 (OK)
numpy >=1.7       :  1.16.3 (OK)
sympy >=0.7.3     :  None (NOK)
cython >=0.21     :  None (NOK)
qtconsole >=4.2.0 :  4.3.1 (OK)
IPython >=4.0     :  5.8.0 (OK)
matplotlib >=2.0.0:  3.0.3 (OK)
pylint >=0.25     :  2.2.2 (OK)
```

**Comment by ccordoba12** [comment]
This is fixed in our latest version (3.3.4). Please update.

**Comment by dittosayan** [comment]
Thanks for the support.



[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
05/17/19,
3:22:38 PM
